libraries/config/include/config/config.hpp

```diff
@@ -46,6 +46,8 @@ struct NetworkConfig {
   uint16_t network_peer_blacklist_timeout = kBlacklistTimeoutDefaultInSeconds;
   bool disable_peer_blacklist = false;
   uint16_t deep_syncing_threshold = 10;
+  uint16_t vote_accepting_rounds = 5;
+  uint16_t vote_accepting_steps = 50;
```

Comment:
Did you try some smaller number ? did it affect the tests ?

Reply:
Accepting only 1 future round didn't worked, so I decided that this values should be ok. It shouldn't be possible to save too much votes with this config. And even this affected some tests that was speeded-up(related to first param in `make_configs` function), for example `FullNodeTest.sync_five_nodes` needed 4x time to complete because it was dropping some votes. But it was passing. So I introduced this thing https://github.com/Taraxa-project/taraxa-node/pull/1957/files#diff-b8695b851212c29d15116200602b07a1de6e81b2e9d49ec2760dc31b7261ddd1R198

Reply:
sure, 1 round cannot work. I think 5 rounds is ok, I am asking about those 50 steps - did you try to use 10, 20, etc... ? If that did not affect the tests I think the less the better

Reply:
I will test it with less steps